code/statsd/src/main/scala/com/avast/datadog4s/statsd/StatsDClient.scala

```diff
@@ -6,15 +6,16 @@ import cats.effect.{ Resource, Sync }
 import com.timgroup.statsd.{ NonBlockingStatsDClient, NonBlockingStatsDClientBuilder }
 
 object StatsDClient {
-  def make[F[_]: Sync](statsDServer: InetSocketAddress, queueSize: Int): Resource[F, NonBlockingStatsDClient] = {
-    val builder = new NonBlockingStatsDClientBuilder()
+  private def makeBuilder(statsDServer: InetSocketAddress, queueSize: Int): NonBlockingStatsDClientBuilder =
+    new NonBlockingStatsDClientBuilder()
       .hostname(statsDServer.getHostName)
       .port(statsDServer.getPort)
       .queueSize(queueSize)
       .prefix("")
-    fromBuilder(builder)
-  }
 
-  def fromBuilder[F[_]: Sync](builder: NonBlockingStatsDClientBuilder): Resource[F, NonBlockingStatsDClient] =
```

Comment:
please make sure that this function still exists in the PR ... for better or worse it once existed and was published as public so there might be users relying on it and i'd prefer not to break the api unless really necessary :)

Reply:
after that i think we are good to merge ;)

Reply:
Ah my bad, restored!